tests/cli_clients.py

```diff
@@ -1,209 +0,0 @@
-from pathlib import Path
```

Comment:
What's going on with the file removal here?

Reply:
It's leftover from early development of this tool, before we had access to the full simulator. No longer used or needed since the tests all run through the sim now.